src/gtc/gtcpp/gtcpp_codegen.py

```diff
@@ -80,7 +80,7 @@ def visit_AccessorRef(self, accessor_ref: gtcpp.AccessorRef, **kwargs):
             return f"eval({accessor_ref.name}())"
         return (
             f"eval({accessor_ref.name}({offset.i}, {offset.j}, {offset.k}"
-            + "".join(f", {d}" for d in accessor_ref.data_index)
+            + "".join(f", {self.visit(d)}" for d in accessor_ref.data_index)
```

Comment:
Same as above, am I missing something?
```suggestion
            + ", ".join(str(d) for d in accessor_ref.data_index)
```

Reply:
Thanks for checking this one out. It's not quite that: we need the comma at the beginning as well (if there are `>1` items in the list), so it might as well be prepended to the strings themselves.

Reply:
Oh, I meant to remove this comment. You are absolutely right.